### PR TITLE
john --> master: updated configuration commands for multiple cameras

### PR DESCRIPTION
**Pull request by jhallard**
Added configuration commands for D-Link and Hikvision models, including the XML files needed.

### CODE REVIEW

File: configs/hikvision_ds-2cd2032-I_stream_01.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?><StreamingChannel xmlns="http://www.hikvision.com/ver20/XMLSchema" version="2.0">
+<id>101</id>
+<channelName>Camera 02</channelName>
```
Comment (jcmaslan) [inline configs/hikvision_ds-2cd2032-I_stream_01.xml]: This should be `Channel 01` to match the id `101`

**Comment by jcmaslan** [comment]
LGTM other than the comment to fix Channel name

File: configs/hikvision_ds-2cd2032-I_stream_01.xml (hunk 1 above)
Comment (jhallard) [reply configs/hikvision_ds-2cd2032-I_stream_01.xml]: done 62892bc

**Comment by jhallard** [comment]
I fixed the channel names so I'll go ahead and merge this.

**Pull request merged**